### PR TITLE
readme file was filled

### PR DESCRIPTION
**Pull request by SergiiSV**

### CODE REVIEW

**Review by begemot1987 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,36 @@
-# taxi-service
+ #**Taxi Service**
```
Comment (Nick97-git) [inline README.md]:
```suggestion
 # Taxi Service
```

File: README.md [hunk 2]
```diff
@@ -1 +1,36 @@
...
+**Taxi-service application** - is an application which can provide basic functionality for creation of taxi service.
```
Comment (Nick97-git) [inline README.md]:
```suggestion
It's an application that can provide basic functionality for the creation of a taxi service.
```

File: README.md [hunk 3]
```diff
@@ -1 +1,36 @@
...
+
+**Application provides next opportunities:**
```
Comment (Nick97-git) [inline README.md]:
```suggestion
## Application provides next opportunities:
```
the same with others

File: README.md [hunk 4]
```diff
@@ -1 +1,36 @@
...
+- Authentication service which provide using of application only by registered users;
+- Register new driver or Login service if driver exist;
+- Register new car manufacturer;
+- Register new car;
+- Create relations between car and its manufacturer;
+- Create relations between cars and their drivers;
+- Possibility to see all information about drivers, manufacturers, cars;
+- Possibility to delete unnecessary driver, manufacturer, unused car.
```
Comment (Nick97-git) [inline README.md]:
```suggestion
- Register
- Login
- Add new car manufacturer and car
- See all information about drivers, manufacturers, cars
- Delete unnecessary driver, manufacturer, car
```

File: README.md [hunk 5]
```diff
@@ -1 +1,36 @@
...
+**During development were used next design rules:**
+- N-tier architecture software design principle: program divided on 3 level: controllers, service, DAO;
+- SOLID principles;
+- dependency injection through fields injection.
+
+**There are technology which were used:**
+- Java 11;
+- JDBC;
+- Servlet;
+- Webserver TOMCAT 9.0.50;
```
Comment (Nick97-git) [inline README.md]:
```suggestion
- Tomcat 9.0.50;
```

File: README.md [hunk 6]
```diff
@@ -1 +1,36 @@
...
+- Possibility to see all information about drivers, manufacturers, cars;
+- Possibility to delete unnecessary driver, manufacturer, unused car.
+
+**During development were used next design rules:**
+- N-tier architecture software design principle: program divided on 3 level: controllers, service, DAO;
+- SOLID principles;
+- dependency injection through fields injection.
+
+**There are technology which were used:**
+- Java 11;
```
Comment (Nick97-git) [inline README.md]:
```suggestion
- Java 11
```
the same with others

File: README.md [hunk 7]
```diff
@@ -1 +1,36 @@
...
+- HTML, CSS.
+
+**Required steps before start the application:**
+- install IntelliJ IDEA ultimate version;
+- install MySQL and MySQL workbench. Collect some data that you need in the next step. Use _resources/init_db.sql_ to create schema at database. Pay attention: if you have schema with name "taxi" it will be replaced; 
+- Fulfill URL, USERNAME, PASSWORD, JDBC_DRIVER fields at _java/taxi/util/ConnectionUtil_ class, to make connection to your database.;
+- download Tomcat webserver. Recommended v.9.0.50 or close, but not v.10+.
+- connect and configure Tomcat inside application;
+- download all Maven dependencies at _pom.xml_ file;
+- to Logging information about logins, set correct "absolute path" on the line _File name="LogToFile" fileName="absolute path"_ to the file where to save logs at _src/main/resources/log4j2.xml_ file.
```
Comment (Nick97-git) [inline README.md]: make a numbered list of it

File: README.md [hunk 8]
```diff
@@ -1 +1,36 @@
...
+- JSP;
+- HTML, CSS.
+
+**Required steps before start the application:**
+- install IntelliJ IDEA ultimate version;
+- install MySQL and MySQL workbench. Collect some data that you need in the next step. Use _resources/init_db.sql_ to create schema at database. Pay attention: if you have schema with name "taxi" it will be replaced; 
+- Fulfill URL, USERNAME, PASSWORD, JDBC_DRIVER fields at _java/taxi/util/ConnectionUtil_ class, to make connection to your database.;
+- download Tomcat webserver. Recommended v.9.0.50 or close, but not v.10+.
+- connect and configure Tomcat inside application;
+- download all Maven dependencies at _pom.xml_ file;
```
Comment (Nick97-git) [inline README.md]: you can remove this line

File: README.md [hunk 9]
```diff
@@ -1 +1,36 @@
...
+- Register new car;
+- Create relations between car and its manufacturer;
+- Create relations between cars and their drivers;
+- Possibility to see all information about drivers, manufacturers, cars;
+- Possibility to delete unnecessary driver, manufacturer, unused car.
+
+**During development were used next design rules:**
+- N-tier architecture software design principle: program divided on 3 level: controllers, service, DAO;
+- SOLID principles;
+- dependency injection through fields injection.
```
Comment (Nick97-git) [inline README.md]:
```suggestion
- Dependency injection 
```

**Review by Nick97-git — CHANGES_REQUESTED** [review]
- I recommend install plugin Grammarly to eliminate some English errors 
- remove redundant `/` from urls like /cars/, /drivers/, /manufacturers/
- LoginController -> driverName -> login
- DeleteCarController, DeleteDriverController, DeleteManufacturerController -> remove /all from /cars/all, /drivers/all, /manufacturers/all url patterns

File: README.md [hunk 10]
```diff
@@ -1 +1,32 @@
...
+## During development were used next design rules:
+- N-tier architecture software design principle: program divided into 3 levels: controllers, service, DAO
+- SOLID principles
+- Dependency injection
+
+## There are technologies that were used:
+- Java 11
+- JDBC
+- Servlet
+- TOMCAT 9.0.50
```
Comment (Nick97-git) [inline README.md:20]:
```suggestion
- Tomcat 9.0.50
```

File: README.md [hunk 11]
```diff
@@ -1 +1,32 @@
-# taxi-service
+# Taxi Service
+It's an application that can provide basic functionality for the creation of taxi service.
+
+## Application provides next opportunities:
+- Register
+- Login
+- Add new car manufacturer and car
```
Comment (Nick97-git) [inline README.md:7]:
```suggestion
- Add new manufacturer and car
```

**Review by Nick97-git — APPROVED** [review]

**Pull request merged**